src/request.ts

```diff
@@ -0,0 +1,9 @@
+// @ts-ignore
+import * as fetch from 'node-fetch';
+import * as ProxyAgent from 'proxy-agent';
+
+export default (uri: string, opts: any = {}) => {
+  const options = process.env.PROXY ? { ...opts, agent: new ProxyAgent(process.env.PROXY) } : opts;
```

Comment:
`process.env.PROXY` may throw a reference error in certain environments.

```suggestion
  const options = typeof process === 'object' && process.env.PROXY ? { ...opts, agent: new ProxyAgent(process.env.PROXY) } : opts;
```